src/Builder/FrontendDialectTransformer.cpp

```diff
@@ -406,7 +494,9 @@ class FrontendGenImpl {
     // the generic operator is used
     // one known reeason is the optional input
 
-    (this->*(import_handler_map_[opName.str()]))(node);
+    auto found = import_handler_map_.find(opName.str());
+    assert(found != import_handler_map_.end() && "Could not find op importer");
```

Comment:
We ran into the case where this just faulted and tracking it down was tricky - which is why we added this code.

Reply:
What corner case made this code error? Missing operation?

Reply:
IIRC the case was that during development we had added a custom ImportNodeSlice (and the corresponding code in gen_onnx_mlir.py) but had accidentally commented the function itself out, which caused a segfault.

Reply:
I'm ok if this leads to a better error message which it looks to me like it does. @tjingrant